src/input/mod.rs

```diff
@@ -2588,6 +2648,12 @@ fn find_bind(
             repeat: true,
             cooldown: None,
             allow_when_locked: false,
+            // In a worst-case scenario, the user has no way to unlock the compositor and a
```

Comment:
Speaking of, I think ToggleInhibit should be added to `allowed_when_locked()`

Reply:
done